frontend/src/components/TransactionsPerSecond/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useMemo } from "react";
+import { useDispatch } from "react-redux";
+import BigNumber from "bignumber.js";
+import { Icon } from "@stellar/design-system";
+
+import "./styles.scss";
+
+import {
+  Network,
+  ActionStatus,
+  LedgerTransactionHistoryFilterType,
+} from "types";
+import {
+  ledgerTransactionHistoryConfig,
+  networkConfig,
+} from "constants/settings";
+import {
+  VerticalBarChart,
+  TimeRange,
+} from "components/charts/VerticalBarChart";
+import { useRedux } from "hooks/useRedux";
+import { SectionCard } from "components/SectionCard";
+import { AmountInfoCard } from "components/AmountInfoCard";
+import { fetchTransactionsHistoryMonthAction } from "ducks/transactions";
+
+export const TransactionsPerSecond = () => {
+  const { transactions } = useRedux("transactions");
+  const dispatch = useDispatch();
+  const selectedTimeInterval = LedgerTransactionHistoryFilterType["30D"];
+
+  useEffect(() => {
+    dispatch(fetchTransactionsHistoryMonthAction(Network.MAINNET));
+  }, [dispatch]);
+
+  const data = useMemo(() => {
```

Comment:
if you hover over the verticalbar chart entries the dates go backwards. I think it may be because data is sorted in the wrong direction?

![Screen Shot 2022-07-11 at 10 56 49 AM](https://user-images.githubusercontent.com/30449853/178327548-c0a94f78-dc34-4e7a-b548-f96c7879c576.png)